dev_guide/shared_memory.adoc

```diff
@@ -12,75 +12,55 @@ toc::[]
- in a pod do not share their mount namespaces so we use volumes to mount in the same /dev/shm into each container in a pod.
+POSIX shared memory requires that a tmpfs be mounted at `/dev/shm`.
+The containers in a pod do not share their mount namespaces so we use volumes to provide the same `/dev/shm` into each container in a pod.
+The following example shows how to set up POSIX shared memory between two containers.
 
-shared-memory.json
 ====
+.shared-memory.yaml
+[source,yaml]
 ----
-{
-    "apiVersion": "v1beta3",
-    "id": "hello-openshift",
-    "kind": "Pod",
-    "metadata": {
-        "name": "hello-openshift",
-        "labels": {
-            "name": "hello-openshift"
-        }
-    },
-    "spec": {
-        "volumes": [
-            {
-                "name": "dshm",
-                "emptyDir": {
-                    "medium": "Memory"
-                }
-            }
-        ],
-        "containers": [
-            {
-                "image": "kubernetes/pause",
-                "name": "hello-container1",
-                "ports": [
-                    {
-                        "containerPort": 8080,
-                        "hostPort": 6061
-                    }
-                ],
-                "volumeMounts": [
-                    {
-                        "mountPath": "/dev/shm",
-                        "name": "dshm"
-                    }
-                ]
-            },
-            {
-                "image": "kubernetes/pause",
-                "name": "hello-container2",
-                "ports": [
-                    {
-                        "containerPort": 8081,
-                        "hostPort": 6062
-                    }
-                ],
-                "volumeMounts": [
-                    {
-                        "mountPath": "/dev/shm",
-                        "name": "dshm"
-                    }
-                ]
-            }
-        ]
-    }
-}
-
+---
+apiVersion: v1beta3
+id: hello-openshift
+kind: Pod
+metadata:
+  name: hello-openshift
+  labels:
+    name: hello-openshift
+spec:
+  volumes:                          <1>
+    - name: dshm
+      emptyDir:
+        medium: Memory
+  containers:
+    - image: kubernetes/pause
+      name: hello-container1
+      ports:
+        - containerPort: 8080
+          hostPort: 6061
+      volumeMounts:                 <2>
+        - mountPath: /dev/shm
+          name: dshm
+    - image: kubernetes/pause
+      name: hello-container2
+      ports:
+        - containerPort: 8081
+          hostPort: 6062
+      volumeMounts:                 <3>
+        - mountPath: /dev/shm
+          name: dshm
 ----
+<1> specifies the tmpfs volume `dshm`.
+<2> enables POSIX shared memory for `hello-container1` via `dshm`.
+<3> enables POSIX shared memory for `hello-container2` via `dshm`.
 ====
 
-****
-`$ oc create -f shared-memory.json`
-****
+----
+$ oc create -f shared-memory.yaml
+----
```

Comment:
Even if it's fairly obvious, maybe add some brief context before this command, e.g.:

```
Create the pod using the *_shared-memory.yaml_* file:
```